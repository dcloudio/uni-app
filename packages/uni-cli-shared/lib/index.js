const tags = require('./tags')

const {
  getJson,
  parseJson
} = require('./json')

const {
  getH5Options,
  getManifestJson,
  getNetworkTimeout,
  parseManifestJson
} = require('./manifest.js')

const {
  getMainEntry,
  getNVueMainEntry,
  parseEntry,
  parsePages,
  getPagesJson,
  parsePagesJson
} = require('./pages')

const {
  md5,
  hasOwn,
  hasModule,
  hashify,
  camelize,
  hyphenate,
  removeExt,
  normalizePath,
  getComponentName,
  convertStaticStyle,
  getTemplatePath,
  createSource,
  deleteAsset,
  showRunPrompt
} = require('./util')

const {
  getFlexDirection,
  getPlatformProject,
  isSupportSubPackages,
  getPlatforms,
  getPlatformScss,
  getPlatformSass,
  runByHBuilderX,
  isInHBuilderX,
  isInHBuilderXAlpha,
  getPlatformExts,
  getPlatformTarget,
  getShadowCss,
  getPlatformCssVars,
  getPlatformCssnano,
  getShadowTemplate,
  jsPreprocessOptions,
  cssPreprocessOptions,
  htmlPreprocessOptions,
  nvueJsPreprocessOptions,
  nvueCssPreprocessOptions,
  nvueHtmlPreprocessOptions,
  getPlatformGlobal,
  getPlatformStat,
  getPlatformPush,
  getPlatformUniCloud,
  getDevUniConsoleCode
} = require('./platform')

const uts = require('./uts')

const {
  parseTheme,
  initTheme
} = require('./theme')

module.exports = {
  uts,
  md5,
  tags,
  hasOwn,
  getJson,
  parseJson,
  hashify,
  hasModule,
  camelize,
  hyphenate,
  removeExt,
  normalizePath,
  parseEntry,
  parsePages,
  getH5Options,
  getMainEntry,
  getNVueMainEntry,
  getPagesJson,
  getManifestJson,
  getNetworkTimeout,
  runByHBuilderX,
  isInHBuilderX,
  isInHBuilderXAlpha,
  isSupportSubPackages,
  getPlatforms,
  getFlexDirection,
  getPlatformScss,
  getPlatformSass,
  getPlatformExts,
  getPlatformTarget,
  getPlatformProject,
  getShadowCss,
  getPlatformCssVars,
  getPlatformCssnano,
  getShadowTemplate,
  parsePagesJson,
  parseManifestJson,
  getComponentName,
  convertStaticStyle,
  getTemplatePath,
  createSource,
  deleteAsset,
  showRunPrompt,
  jsPreprocessOptions,
  cssPreprocessOptions,
  htmlPreprocessOptions,
  nvueJsPreprocessOptions,
  nvueCssPreprocessOptions,
  nvueHtmlPreprocessOptions,
  getPlatformGlobal,
  getPlatformStat,
  getPlatformPush,
  getPlatformUniCloud,
  parseTheme,
  initTheme,
  getDevUniConsoleCode
}
