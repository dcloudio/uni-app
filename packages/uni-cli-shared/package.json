{
  "name": "@dcloudio/uni-cli-shared",
  "version": "2.0.0-28920200907001",
  "description": "uni-cli-shared",
  "main": "lib/index.js",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/dcloudio/uni-app.git",
    "directory": "packages/uni-cli-shared"
  },
  "files": [
    "lib",
    "template",
    "components"
  ],
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "fxy060608",
  "license": "Apache-2.0",
  "dependencies": {
    "hash-sum": "^1.0.2",
    "postcss-urlrewrite": "^0.2.2",
    "strip-json-comments": "^2.0.1"
  },
  "gitHead": "e51484a49748f7bff86bb833763e5dff1522d380"
}
