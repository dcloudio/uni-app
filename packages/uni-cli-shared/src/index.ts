export * from './fs'
export * from './mp'
export * from './url'
export * from './env'
export * from './hbx'
export * from './ssr'
export * from './vue'
export * from './uts'
export * from './logs'
export * from './i18n'
export * from './deps'
export * from './json'
export * from './vite'
export * from './utils'
export * from './easycom'
export * from './constants'
export * from './preprocess'
export * from './postcss'
export * from './filter'
export * from './esbuild'
export * from './resolve'
export * from './scripts'
export * from './platform'
export * from './utsUtils'

export {
  parseUniExtApi,
  parseUniExtApis,
  parseInjects,
  Define,
  DefineOptions,
  Defines,
  getUniExtApiProviderRegisters,
  formatExtApiProviderName,
} from './uni_modules'
export {
  parseUniModulesArtifacts,
  resolveEncryptUniModule,
} from './uni_modules.cloud'
import type { EncryptPackageJson } from './uni_modules.cloud'
export type EncryptArtifacts = EncryptPackageJson['uni_modules']['artifacts']

export { M } from './messages'

export * from './exports'
export { checkUpdate } from './checkUpdate'
