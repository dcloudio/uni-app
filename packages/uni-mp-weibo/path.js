const path = require('path')
module.exports = {
  src: path.join(__dirname, 'src'),
  lib: path.join(__dirname, 'lib')
}
