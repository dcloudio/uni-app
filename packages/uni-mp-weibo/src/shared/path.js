export function parsePath (path) {
  let hash = ''
  let query = ''

  const hashIndex = path.indexOf('#')
  if (hashIndex >= 0) {
    hash = path.slice(hashIndex)
    path = path.slice(0, hashIndex)
  }

  const queryIndex = path.indexOf('?')
  if (queryIndex >= 0) {
    query = path.slice(queryIndex + 1)
    path = path.slice(0, queryIndex)
  }

  return {
    path,
    query,
    hash
  }
}
