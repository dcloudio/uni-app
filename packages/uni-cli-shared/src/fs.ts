import fs from 'fs'
import path from 'path'

export function emptyDir(dir: string, skip: string[] = []) {
  for (const file of fs.readdirSync(dir)) {
    if (skip.includes(file)) {
      continue
    }
    // node >= 14.14.0
    fs.rmSync(path.resolve(dir, file), { recursive: true, force: true })
  }
}
