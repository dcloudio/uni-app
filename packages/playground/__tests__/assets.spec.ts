import fs from 'fs-extra'
import path from 'path'
import execa from 'execa'
import { sync } from 'fast-glob'
import { isBinaryFileSync } from 'isbinaryfile'
import { normalizePath } from '@dcloudio/uni-cli-shared'

const projectDir = normalizePath(path.resolve(__dirname, '../assets'))

describe('assets playground', () => {
  jest.setTimeout(50 * 1000)
  const types = {
    'uni-app': [
      // dev 目前面临需要exit的问题
      // 'dev:app',
      // 'dev:mp-alipay',
      // 'dev:mp-baidu',
      // 'dev:mp-kuaishou',
      // 'dev:mp-lark',
      // 'dev:mp-qq',
      // 'dev:mp-toutiao',
      // 'dev:mp-weixin',
      'build:app',
      'build:h5',
      'build:mp-alipay',
      'build:mp-baidu',
      'build:mp-kuaishou',
      'build:mp-lark',
      'build:mp-qq',
      'build:mp-toutiao',
      'build:mp-weixin',
    ],
    'uni-app-x': [
      'dev:app-android',
      'dev:app-ios',
      'build:app-android',
      'build:app-ios',
      'build:h5',
      //   'build:mp-alipay',
      //   'build:mp-baidu',
      //   'build:mp-kuaishou',
      //   'build:mp-lark',
      //   'build:mp-qq',
      //   'build:mp-toutiao',
      'build:mp-weixin',
    ],
  }
  const distDir = path.resolve(projectDir, 'dist')
  if (fs.existsSync(distDir)) {
    fs.emptyDirSync(distDir)
  }
  Object.keys(types).forEach((type) => {
    const scripts = types[type]
    scripts.forEach((script) => {
      const mode = script.split(':')[0]
      const platform = script.split(':')[1]
      test(`${type} ${script}`, async () => {
        const outDir = path.resolve(distDir, mode, type, platform)
        console.log(`npm run ${script} start`)
        await execa('npm', ['run', script], {
          cwd: projectDir,
          env: {
            ...process.env,
            UNI_OUTPUT_DIR: outDir,
            UNI_APP_X: type === 'uni-app-x' ? 'true' : 'false',
          },
        })
        console.log(`npm run ${script} end`)
        sync('**/*', { cwd: outDir, absolute: true, dot: true })
          .sort()
          .forEach((file) => {
            const basename = path.basename(file)
            if (basename.startsWith('__uniapp')) {
              return
            }
            let relativePath = normalizePath(path.relative(outDir, file))
            if (isBinaryFileSync(file)) {
              expect(path.basename(file)).toMatchSnapshot(relativePath)
            } else {
              const content = fs.readFileSync(file, 'utf8')
              const matches = content.match(/"([^"]*\.(png|jpg))"/g)
              if (matches) {
                const webPageJsArr = [
                  'assets/pages-index-index.',
                  'assets/pages-sub-test-test.',
                ]
                // 避免hash影响
                webPageJsArr.forEach((pageJs) => {
                  if (relativePath.startsWith(pageJs)) {
                    relativePath = pageJs + 'js'
                  }
                })
                expect(matches).toMatchSnapshot(relativePath)
              }
            }
          })
      })
    })
  })
})
