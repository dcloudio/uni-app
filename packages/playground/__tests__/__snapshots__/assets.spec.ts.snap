// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`assets playground uni-app build:app: app-service.js 1`] = `
[
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
]
`;

exports[`assets playground uni-app build:app: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app build:app: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app build:app: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app build:app: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app build:app: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app build:h5: assets/pages-index-index.js 1`] = `
[
  ""/assets/uni_modules-test-com-static_shuijiao-B0f5FEMg.jpg"",
  ""/assets/pages-index-static_invalid-static-CONWi2JP.jpg"",
]
`;

exports[`assets playground uni-app build:h5: assets/pages-index-static_invalid-static-CONWi2JP.jpg 1`] = `"pages-index-static_invalid-static-CONWi2JP.jpg"`;

exports[`assets playground uni-app build:h5: assets/pages-sub-static_2-DQMwIfWk.jpg 1`] = `"pages-sub-static_2-DQMwIfWk.jpg"`;

exports[`assets playground uni-app build:h5: assets/pages-sub-test-test.js 1`] = `
[
  ""/assets/pages-sub-static_2-DQMwIfWk.jpg"",
]
`;

exports[`assets playground uni-app build:h5: assets/uni_modules-test-com-static_shuijiao-B0f5FEMg.jpg 1`] = `"uni_modules-test-com-static_shuijiao-B0f5FEMg.jpg"`;

exports[`assets playground uni-app build:h5: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app build:h5: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app build:h5: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app build:h5: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app build:mp-alipay: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app build:mp-alipay: common/assets.js 1`] = `
[
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
]
`;

exports[`assets playground uni-app build:mp-alipay: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app build:mp-alipay: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app build:mp-alipay: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app build:mp-alipay: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app build:mp-baidu: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app build:mp-baidu: common/assets.js 1`] = `
[
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
]
`;

exports[`assets playground uni-app build:mp-baidu: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app build:mp-baidu: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app build:mp-baidu: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app build:mp-baidu: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app build:mp-kuaishou: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app build:mp-kuaishou: common/assets.js 1`] = `
[
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
]
`;

exports[`assets playground uni-app build:mp-kuaishou: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app build:mp-kuaishou: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app build:mp-kuaishou: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app build:mp-kuaishou: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app build:mp-lark: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app build:mp-lark: common/assets.js 1`] = `
[
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
]
`;

exports[`assets playground uni-app build:mp-lark: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app build:mp-lark: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app build:mp-lark: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app build:mp-lark: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app build:mp-qq: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app build:mp-qq: common/assets.js 1`] = `
[
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
]
`;

exports[`assets playground uni-app build:mp-qq: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app build:mp-qq: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app build:mp-qq: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app build:mp-qq: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app build:mp-toutiao: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app build:mp-toutiao: common/assets.js 1`] = `
[
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
]
`;

exports[`assets playground uni-app build:mp-toutiao: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app build:mp-toutiao: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app build:mp-toutiao: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app build:mp-toutiao: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app build:mp-weixin: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app build:mp-weixin: common/assets.js 1`] = `
[
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
]
`;

exports[`assets playground uni-app build:mp-weixin: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app build:mp-weixin: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app build:mp-weixin: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app build:mp-weixin: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app-x build:app-android: .uniappx/android/src/index.kt 1`] = `
[
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
]
`;

exports[`assets playground uni-app-x build:app-android: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app-x build:app-android: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app-x build:app-android: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app-x build:app-android: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app-x build:app-android: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app-x build:app-ios: app-service.js 1`] = `
[
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
]
`;

exports[`assets playground uni-app-x build:app-ios: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app-x build:app-ios: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app-x build:app-ios: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app-x build:app-ios: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app-x build:app-ios: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app-x build:h5: assets/pages-index-index.js 1`] = `
[
  ""/assets/uni_modules-test-com-static_shuijiao-B0f5FEMg.jpg"",
  ""/assets/pages-index-static_invalid-static-CONWi2JP.jpg"",
]
`;

exports[`assets playground uni-app-x build:h5: assets/pages-index-static_invalid-static-CONWi2JP.jpg 1`] = `"pages-index-static_invalid-static-CONWi2JP.jpg"`;

exports[`assets playground uni-app-x build:h5: assets/pages-sub-static_2-DQMwIfWk.jpg 1`] = `"pages-sub-static_2-DQMwIfWk.jpg"`;

exports[`assets playground uni-app-x build:h5: assets/pages-sub-test-test.js 1`] = `
[
  ""/assets/pages-sub-static_2-DQMwIfWk.jpg"",
]
`;

exports[`assets playground uni-app-x build:h5: assets/uni_modules-test-com-static_shuijiao-B0f5FEMg.jpg 1`] = `"uni_modules-test-com-static_shuijiao-B0f5FEMg.jpg"`;

exports[`assets playground uni-app-x build:h5: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app-x build:h5: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app-x build:h5: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app-x build:h5: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app-x build:mp-weixin: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app-x build:mp-weixin: common/assets.js 1`] = `
[
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
]
`;

exports[`assets playground uni-app-x build:mp-weixin: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app-x build:mp-weixin: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app-x build:mp-weixin: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app-x build:mp-weixin: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app-x dev:app-android: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app-x dev:app-android: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app-x dev:app-android: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app-x dev:app-android: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app-x dev:app-android: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;

exports[`assets playground uni-app-x dev:app-ios: app-service.js 1`] = `
[
  ""/uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg"",
  ""/static/static_logo.png"",
  ""/assets/pages-index-static_invalid-static.34ae5d3b.jpg"",
  ""/static/static_logo.png"",
  ""/pages/sub/static/pages-sub-static_2.jpg"",
]
`;

exports[`assets playground uni-app-x dev:app-ios: assets/pages-index-static_invalid-static.34ae5d3b.jpg 1`] = `"pages-index-static_invalid-static.34ae5d3b.jpg"`;

exports[`assets playground uni-app-x dev:app-ios: pages/sub/static/pages-sub-static_2.jpg 1`] = `"pages-sub-static_2.jpg"`;

exports[`assets playground uni-app-x dev:app-ios: static/static_logo.png 1`] = `"static_logo.png"`;

exports[`assets playground uni-app-x dev:app-ios: uni_modules/test-com/static/uni_modules-test-com-static_logo.png 1`] = `"uni_modules-test-com-static_logo.png"`;

exports[`assets playground uni-app-x dev:app-ios: uni_modules/test-com/static/uni_modules-test-com-static_shuijiao.jpg 1`] = `"uni_modules-test-com-static_shuijiao.jpg"`;
